docs/insights/coverage.md

```diff
@@ -0,0 +1,67 @@
+---
+id: coverage
+title: Measuring Your Test Coverage
+sidebar_label: Coverage
+description: See a breakdown of the browser and device variations covered by your tests.
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+The Coverage report allows Org Admins to view and download the breakdown of browsers and mobile devices the tests across their organization have run against. Having insight into your coverage data helps companies:
+
+* See at a glance whether their testing is in sync with the environments in which it is most often accessed
+* Design their test strategy to keep up with newly released browser and device versions
+* Utilize their concurrency to spread testing across test environments for greater efficiency and comprehensiveness
+
+:::note Org Admins Only
+You must have [Org Admin](/basics/acct-team-mgmt/managing-user-info/#user-roles) privileges for your organization in order to view and download coverage data.
+:::
+
+## Access the Coverage Reports
+
+1. Click the **Insights** tab to expand its submenu.
+1. Click **Coverage** (only visible to Org Admin users).
+1. Specify the type of coverage you wish to view by selecting the **Devices** or **Browsers** tab.
+1. Specify whether you wish to see coverage data for **Virtual Cloud** tests (desktop web-apps, mobile emulators and simulators) or **Real Devices** (physical devices located in our data centers and connected to our platform).
+1.Use the [filters](scope.md#using-filters-to-adjust-the-scope-of-your-data) to show results for a particular user's tests and/or a specific time period.
```

Comment:
Add space after **1.**

Apply the **Owner** and time period [filters](scope.md#using-filters-to-adjust-the-scope-of-your-data) to show data for a particular user's tests and/or a specific time period.